C#/Botkit/Microsoft.BotKit.Adapters.Slack/SlackAdapter.cs

```diff
@@ -285,8 +301,8 @@ public override async Task DeleteActivityAsync(ITurnContext turnContext, Convers
             {
                 try
```

Comment:
The same applies here. Remove the `try/catch` blocks, as the exceptions should be handled in those methods which call the actual one.